Formula/moco.rb

```diff
@@ -1,15 +1,15 @@
 class Moco < Formula
   desc "Stub server with Maven, Gradle, Scala, and shell integration"
   homepage "https://github.com/dreamhead/moco"
-  url "https://search.maven.org/remotecontent?filepath=com/github/dreamhead/moco-runner/0.10.2/moco-runner-0.10.2-standalone.jar"
-  version "0.10.2"
-  sha256 "ef946d090d3108843708c194809c57f192e3623dbbbcf86bdee54ce93c299a41"
+  url "https://repo1.maven.org/maven2/com/github/dreamhead/moco-runner/0.11.0/moco-runner-0.11.0-standalone.jar"
```

Comment:
A few very simple reasons:

- It was used before in this formula.
- We use it for the majority of our formulae that we fetch from maven.org.
- It will redirect to an appropriate download server and thus probably better distribute the load. (And it is documented in [Maven Central's API](https://search.maven.org/#api).)
- central.maven.org has the disadvantage that it's not accessible via HTTPS.

That said, I have no definite resource that I can quote. What are your reasons to move away from the previously used URL in this formula? (If you have good reasons, I'd like to hear them.)